packaging/nnstreamer.spec

```diff
@@ -1243,6 +1268,14 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_tvm.so
 %endif
 
+%if 0%{?tizen_hal_support}
+%files tizen-hal
+%manifest nnstreamer.manifest
+%defattr(-,root,root,-)
+%{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_snpe.so
```

Comment:
plz consider checking dependency by snpe_support and vivante_support in spec file.

Reply:
my fault, it is unnecesssary, right?

Reply:
Yes tizen-hal package should be built for every configuration